content/sigs/platform/index.adoc

```diff
@@ -51,9 +51,7 @@ The group focuses on technologies used in Jenkins:
 * Architectures and platforms
 * Web Containers
 
-Platform SIG may be cooperating with other groups.
-For example, we will be cooperating with the link:/sigs/cloud-native[Cloud-Native Jenkins group]
```

Comment:
I even did not know this one existed. 😨

Reply:
It has been inactive for a very long time.  One of my cleanup tasks from the governance board is to remove references to inactive special interest groups.